### ISSUE TITLE
activity-translations to use weblate.sugarlabs.org

### ISSUE THREAD

**Issue by quozl** [sugarlabs/sugar-tools#12]

**Comment by chimosky** [comment]
weblate.sugarlabs.org automatically pushes changes to GH, only thing we need to setup is a workflow that automatically opens a PR from the weblate-sync branch - the branch weblate pushes to - to the master/main branch so we'll not need to think about updates from weblate in the activity release process.

**Comment by quozl** [comment]
That did not happen for the Read activity.  Why not?

**Comment by chimosky** [comment]
Seems I'd forgotten to add the webhook for it, and I just did.

Also there's been no change in the activity translation on weblate, so weblate didn't push anything, your recent changes have been pulled in by weblate and if any changes occur there, weblate will auto-update the repo on GH.

**Comment by quozl** [comment]
A summary of the workflow would be great.  Perhaps in the sugar-docs/contributing.md

**Comment by chimosky** [comment]
Yes, there's a summary on the [wiki](https://wiki.sugarlabs.org/go/Service/Weblate), I'll add a reference to it in sugar-docs.
